src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,99 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIALIZATION_LENGTH = 10;
+    private static final String ERROR_MESSAGE_INDEX = "Your index is incorrect";
+    private static final String ERROR_MESSAGE_ELEMENT = "Can't remove element for such";
+    private static final double GROW_INDEX = 1.5;
+    protected T[] array;
+    protected int size = 0;
```

Comment:
why protected?

Reply:
When it was private you said "Let’s add the correct access modifiers". I don`t understand so which it must be.